data/DSLAB/projects.yaml

```diff
@@ -189,9 +234,30 @@ projects:
         tags:
             - Cloud
             - Performance
+        information:
+          - type: Paper
+            title: "Parallel Symbolic Execution for Automated Real-World Software Testing"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "EuroSys 2011"
+                url: http://eurosys2011.cs.uni-salzburg.at/pdf/eurosys2011-salomie.pdf
+          - type: Paper
+            title: "Automated Software Testing as a Service"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "SoCC 2010"
```

Comment:
... and the URL for the conf is: https://dl.acm.org/doi/proceedings/10.1145/1807128
(it is in the "Benchmarking & testing" session)